### ISSUE TITLE
dip provision fails if there is a space in the path name

### ISSUE THREAD

**Issue by unikitty37** [bibendi/dip#130]
**Describe the bug**
When there is a space in the path to the current working folder, `dip provision` fails.

**To Reproduce**
Steps to reproduce the behaviour:

1. Create a folder `/tmp/Test Projects/project1`.
2. Create a basic `dip.yml` in the folder:

```yaml
version: '4.1'

compose:
  files:
    - docker-compose.yml
  project_name: test

provision:
  - dip compose down
  - dip compose up -d
```

3. Create a basic `docker-compose.yml` in the folder:

```yaml
version: '3.5'

services:
  test:
    image: alpine
    command: tail -f /dev/null
```

4. Run `dip provision` from the folder.

**Expected behaviour**
The container is provisioned without error.

**Actual behaviour**
The following error is emitted:

```
No such command: Projects/project1/docker_compose.yml
```

**Screenshots**

**Context (please complete the following information):**
 - OS: macOS 10.14.6 (18G9028)
 - Version: 6.0.0

**Additional context**
The error is doubly confusing as the file specified is `docker-compose.yml`, with a hyphen, and yet the error message mentions `docker_compose.yml` with an underscore.

**Comment by dshramko** [comment]
I think this can solve the issue:
Shellwords module - [escape](https://ruby-doc.org/stdlib-2.5.3/libdoc/shellwords/rdoc/Shellwords.html#method-c-shellescape)

**Comment by bibendi** [comment]
Fixed at v7.1.4
